Makefile

```diff
@@ -323,6 +326,8 @@ UK_ALIBS:=
 UK_ALIBS-y:=
 UK_OLIBS:=
 UK_OLIBS-y:=
+UK_DRIV:=
+UK_DRIV-y:=
```

Comment:
I don't think that we need a `UK_DRIV` list in the build system. Drivers would use the common library registration methods and drivers will appear under `UK_OLIBS`